tests/tools/commands/test_venv.py

```diff
@@ -798,3 +799,65 @@ def create_venv_pex():
     assert os.path.exists(venv_dir)
     assert not os.path.exists(venv_pex)
     assert not os.path.exists(pex_root)
+
+
+@pytest.mark.parametrize(
+    "enable_system_site_package", [pytest.param(flag, id=str(flag)) for flag in [True, False]]
+)
+def test_system_site_package(
+    tmpdir,
+    enable_system_site_package,  # type: bool
+):
+    # type: (...) -> None
+    pex_root = os.path.join(str(tmpdir), "pex_root")
+
+    venv_pex = os.path.join(str(tmpdir), "venv.pex")
+    run_pex_command(
+        args=[
+            "--pex-root",
+            pex_root,
+            "--runtime-pex-root",
+            pex_root,
+            "-o",
+            venv_pex,
+            "--include-tools",
+        ]
+    ).assert_success()
+
+    venv_dir = os.path.join(str(tmpdir), "venv_dir")
+    assert not os.path.exists(venv_dir)
+    create_venv_args = [venv_pex, "venv", venv_dir]
+    if enable_system_site_package:
+        create_venv_args.append("--system-site-packages")
+    subprocess.check_call(args=create_venv_args, env=make_env(PEX_TOOLS=True))
+
+    assert os.path.exists(venv_dir)
+    assert os.path.exists(venv_pex)
+    assert os.path.exists(pex_root)
+
+    # Check pyvenv.cfg
+    pyvenv_cfg_lines = open(os.path.join(venv_dir, "pyvenv.cfg")).readlines()
```

Comment:
This test is failing for Python 2.7 / Virtualenv . Since its besides the point, you only really care about what the tests below prove, how about eliminate this test and just keep the set math tests.

Reply:
Python 2.7 was deprecated, why do we need to test for python 2.7? If that test is required, I'm afraid "pyvenv.cfg" magic doesn't apply to python 2.7, so maybe we should skip this test on python < 3. Let me check the python doc.

Reply:
Pex supports Python 2.7 and 3.5+

Please just remove that part of the test. Like I said, it doesn't test anything you actually care about anyway. What you care about is the site packages set math.

Reply:
Python 2.7 doesn't support "pyvenv.cfg" according to https://docs.python.org/2.7/library/site.html .

Reply:
OK I removed pyvenv.cfg check. I'm afraid the test will fail on py2.7. Could you please let the CI run?